aqua/qt6/Portfile

```diff
@@ -256,8 +257,8 @@ array set modules {
         {"Qt 3D"}
         ""
         "variant overrides: "
-        "revision 0"
-        "License: "
+        "revision 1"
+        "License: GPL-2+"
```

Comment:
I have not confirmed this is the correct license, although it seems that it could be. Can a comment please be added with link or brief explanation for the unusual module license?

Reply:
It's funny. I had to add this to get the install process to continue.  The error out of the install was something to the effect that this license was expected and did not match the null value previously there.

When I get some time, I'll do a little more research and add the comment.

Reply:
Strangely, everything compiled and installed without GPL-2+ when I just tested it.  I'll back it out and see if the CI chain reports anything.

Reply:
the CI chain for macos-11 managed to successfully build qt3d without the license mod (the others times out / were cancelled before getting there).

I think it's safe to leave unmoddified.